cmd/influxd/run.go

```diff
@@ -297,12 +292,14 @@ func openServer(config *Config, b *influxdb.Broker, initializing, configExists b
 }
 
 // initializes a new server that does not yet have an ID.
-func initializeServer(u *url.URL, s *influxdb.Server, b *influxdb.Broker, w io.Writer) {
+func initializeServer(u *url.URL, s *influxdb.Server, b *influxdb.Broker, w io.Writer, initBroker bool) {
 	// TODO: Create replica using the messaging client.
 
-	// Create replica on broker.
-	if err := b.CreateReplica(1, u); err != nil {
-		log.Fatalf("replica creation error: %s", err)
+	if initBroker {
```

Comment:
A server needs to create replicas on the broker regardless of whether this node is initializing as a broker. In other words, this won't work if the node is running as a server *only*.